packages/ng/applications/sandbox/src/app/issues/establishment/establishment.component.html

```diff
@@ -4,21 +4,29 @@ <h2>establishment pager operator</h2>
 	<lu-select class="textfield-input" [(ngModel)]="item">
 		<ng-container *luDisplayer="let value">{{value.name}}</ng-container>
 		<lu-option-picker-advanced>
-			<lu-establishment-pager [filters]="filters"></lu-establishment-pager>
+			<lu-establishment-pager></lu-establishment-pager>
 			<lu-option *luForOptions="let establishment; trackBy: trackBy" [value]="establishment">{{establishment.name}}</lu-option>
 		</lu-option-picker-advanced>
 	</lu-select>
 	<label for="" class="textfield-label">establishment</label>
 </div>
 
 <h2>all in one directive</h2>
+
 <div class="textfield">
 	<lu-establishment-select class="textfield-input" [(ngModel)]="item"></lu-establishment-select>
 	<label for="" class="textfield-label">establishment select</label>
 </div>
-<code>{{item | json }}</code>
+<code><pre>{{item | json }}</pre></code>
+
+<div class="textfield">
+	<lu-establishment-select class="textfield-input" [(ngModel)]="item2" [establishmentDisplayer]="customDisplayer"></lu-establishment-select>
```

Comment:
nope on the customDisplayer

Reply:
Okay but PopleeRem will eventually want to reproduce this kind of display: 

![image](https://user-images.githubusercontent.com/8553617/107771547-308d5980-6d3b-11eb-8613-42618d21424a.png)

(they display the currency next to the establishment's name)

How can I allow them to achieve this without recreating the whole establishment-picker?

Reply:
v1 - dont do it, no complex optim _a priori_
v2 - something similar to the user-select (see [here](https://github.com/LuccaSA/lucca-front/blob/master/packages/ng/libraries/user/src/lib/select/input/user-select-input.component.html#L27)) or or or if the only relevant additional info displayed is the currency (which it might be) add `@Input() displayCurrency`
v3 - with the whole rigamarole around the rework of select-multiple - find an elegant and standard way to do it and apply it to all future selects

Reply:
The currency is not available on the establishments API 😐

Reply:
wot ? but the currency is defined at establishment level in lucca

Reply:
No it isn't : it's defined by the country the legal unit belongs to